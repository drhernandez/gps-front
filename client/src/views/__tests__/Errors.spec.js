import React from 'react';
import renderer from 'react-test-renderer';

import Errors from "../Errors"

describe('Errors view', () => {
  
  it("snapshot renders", () => {

    const component = renderer.create(
      <Errors />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});