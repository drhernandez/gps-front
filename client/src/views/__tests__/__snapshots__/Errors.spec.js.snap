// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors view snapshot renders 1`] = `
<div
  className="main-content-container px-4 pb-4 container-fluid"
>
  <div
    className="error"
  >
    <div
      className="error__content"
    >
      <h2>
        500
      </h2>
      <h3>
        Something went wrong!
      </h3>
      <p>
        There was a problem on our end. Please try again later.
      </p>
      <button
        className="btn btn-primary btn-pill"
        onClick={[Function]}
      >
        ‚Üê Go Back
      </button>
    </div>
  </div>
</div>
`;
